test/Window/Window.test.cpp

```diff
@@ -28,4 +28,26 @@ TEST_CASE("[Window] sf::Window", runDisplayTests())
                                 sf::ContextSettings());
         CHECK(window.getSize() == sf::Vector2u(256, 256));
     }
+
+    SECTION("Move construction")
+    {
+        sf::Window       movedWindow(sf::VideoMode(sf::Vector2u(256, 256), 32),
```

Comment:
Maybe the video mode size could be defined as `{256, 256}` to be consistent with what is done in `WindowBase.test.cpp`? If we do so, other test cases in `Window.test.cpp` would be changed too.